float64 Schutzzeit
float64 Stopzeit
---